src/OpenColorIO/GpuShaderUtils.cpp

```diff
@@ -222,6 +222,28 @@ std::string getMatrixValues(const T * mtx, GpuLanguage lang, bool transpose)
     return vals;
 }
 
+
+void RGBtoRGBATexture(const float* lutValues, int valueCount, std::vector<float>& float4AdaptedLutValues){
```

Comment:
Minor, please add the opening { on a newline. Until we get clang-format!

Reply:
Sure. Fixed this one and other '{'s in a later commit